app/javascript/css/tailwind.scss

```diff
@@ -213,15 +213,15 @@ a {
 }
 
 .button {
-    @apply bg-grey-dark text-white ml-4 py-2 px-3 no-underline ;
+    @apply bg-grey-darker text-white font-bold py-2 px-5 no-underline ;
```

Comment:
this is used elsewhere - please check and create a new class if required

Reply:
Changed this back to the previous code